lib/routes/shu/xykd.ts

```diff
@@ -0,0 +1,95 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio'; // cheerio@1.0.0
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+
+const notice_type = {
+    whxx: { title: '上海大学 - 文化信息', url: 'https://www.shu.edu.cn/xnrc/whxx.htm' },
+    xsbg: { title: '上海大学 - 学术报告', url: 'https://www.shu.edu.cn/xnrc/xsbg.htm' },
+};
+
+export const route: Route = {
+    path: '/xykd/:type?',
+    categories: ['university'],
+    example: '/shu/xykd/xsbg',
+    parameters: { type: '分类，默认为学术公告' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['www.shu.edu.cn/'],
+            target: '/xykd',
+        },
+    ],
+    name: '官网的校园看点',
+    maintainers: ['GhhG123'],
+    handler,
+    url: 'www.shu.edu.cn/',
+    description: `| 文化信息 | 学术报告 |
+  | -------- | --------- |
+  | whxx     | xsbg      |`,
+};
+
+async function handler(ctx) {
+    const type = ctx.req.param('type') ?? 'xsbg';
+    const rootUrl = 'https://www.shu.edu.cn';
+
+    // 发起 HTTP GET 请求
+    const response = await got({
```

Comment:
Use either https://github.com/DIYgod/RSSHub/blob/8b48453ed55165a9e2adbda0705bb8bf6accd1e1/lib/routes/shu/xykd.ts#L46-L54 or https://github.com/DIYgod/RSSHub/blob/8b48453ed55165a9e2adbda0705bb8bf6accd1e1/lib/routes/shu/xykd.ts#L79 but not both style